configs/examples/gift-adversary-experiment.cfg

```diff
@@ -0,0 +1,12 @@
+@include ../active-experiment.cfg
```

Comment:
I also wonder if this is the best directory for the file? It feels a bit lonely as the only `*.cfg` in a sea of `*.json`s in `config/examples`. But to be fair I can't think of a better one -- doesn't belong at the top-level.